docs/introduction/tasks.md

```diff
@@ -1,148 +1,238 @@
 # Tasks
 
-The **Inductiva API** client provides the ability to run complex simulations and
-contains easy-to-use tools to manage them. Each simulation was submitted - with
-`simulator.run()` - runs asynchronously and returns a `Task` object that contains
-methods to manage the simulation.
+At the heart of the API lies the concept of a `Task`.
+A task is an abstraction that encapsulates all the information about the
+computational workload generated by the user. A task is generated when a
+simulation is submitted to the API and exists until the user deletes it.
+With a task at hand, the user can manage the simulation, retrieve its status,
```

Comment:
Not sure if we should address it as *T*ask. We are doing the same for other core concepts, such as "Machine Groups"